src/components/UserProfile/ProfileEvents/ProfileEvents.tsx

```
@@ -0,0 +1,156 @@
+      </div>
+    );
+  }
+
+  if (loading) {
+    return (
+      <div
+        className="d-flex justify-content-center align-items-center"
+        style={{ minHeight: '400px' }}
+        aria-live="polite"
+      >
+        <Spinner animation="border" role="status">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)